luigi/rpc.py

```diff
@@ -71,7 +72,7 @@ def _async_fetch(self, full_url, body):
 
     def _fetch(self, url, body, log_exceptions=True, attempts=3):
 
-        full_url = 'http://%s:%d%s' % (self._host, self._port, url)
+        full_url = 'http://%s:%d%s%s' % (self._host, self._port, self._url_prefix, url)
```

Comment:
While you're at it, you maybe want to change this to use `.format()`. :)

(I thought `%` syntax was deprecated in python 3. No?)